### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by KongkaMozumderSUST**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by sohansai — COMMENTED** [review]
Your contribution is not valid

The title doesn't match the provided link

Read 'docs/CONTRIBUTING.md' before making contributions

**Comment by sohansai** [comment]
also, please keep changes on this PR. don't close it and start a new one. It's not respectful to the reviewers because it erases the reviews.

**Comment by Ajay-singh1** [comment]
@KongkaMozumderSUST Fix the linter errors.

**Comment by Ajay-singh1** [comment]
Looks good to me!

File: books/free-programming-books-bn.md [hunk 1]
```diff
@@ -17,6 +17,9 @@
 
 * [Dynamic Programming Book «ডাইনামিক প্রোগ্রামিং বই»](https://dp-bn.github.io) - Tasmeem Reza, Mamnoon Siam (PDF, [LaTeX](https://github.com/Bruteforceman/dynamic-progamming-book))
 
+* [Number Theory, Graph Theory, LinkedList in Bangla] (https://progkriya.org) (HTML)
```
Comment (eshellman) [inline books/free-programming-books-bn.md:20]: not a book

Comment (eshellman) [reply books/free-programming-books-bn.md:20]:
```suggestion
* [Number Theory, Graph Theory, LinkedList in Bangla](https://progkriya.org) (HTML)
```

Comment (eshellman) [reply books/free-programming-books-bn.md:20]: Don't invent titles! Use the title given by the resource.

File: books/free-programming-books-bn.md [hunk 2]
```diff
@@ -17,6 +17,9 @@
...
+* [Coin Change, Knapsack, Dp](https://www.shafaetsplanet.com) (HTML)
```
Comment (eshellman) [inline books/free-programming-books-bn.md:21]: not a book
Don't invent titles! Use the title given by the resource.

**Review by eshellman — COMMENTED** [review]
no books to add

**Pull request closed without merging**